Library/Formula/imgur-screenshot.rb

```diff
@@ -0,0 +1,23 @@
+class ImgurScreenshot < Formula
+  desc "Take screenshot selection, upload to imgur. + more cool things"
+  homepage "https://github.com/jomo/imgur-screenshot"
+  head "https://github.com/jomo/imgur-screenshot.git"
+  url "https://github.com/jomo/imgur-screenshot/archive/v1.5.4.tar.gz"
+  sha256 "147fe1c3b3e407d952fa8abbdb9b436192133cf7a56278edc0dd87c2a9c9c430"
+
+  option "with-terminal-notifier", "terminal-notifier is needed for Mac OS X User Notifications"
+  depends_on "terminal-notifier" => :optional
+
+  def install
+    bin.install "imgur-screenshot.sh"
```

Comment:
Nope. If upstream recommend people use it with the `.sh` extension intact, we should respect that.